src/coreclr/src/tools/ILVerification/ILVerification.csproj

```diff
@@ -4,300 +4,12 @@
-    </Compile>
-    <Compile Include="..\Common\TypeSystem\Ecma\MetadataExtensions.cs">
-      <Link>Ecma\MetadataExtensions.cs</Link>
-    </Compile>
-    <Compile Include="..\Common\TypeSystem\Ecma\IMetadataStringDecoderProvider.cs">
-      <Link>Ecma\IMetadataStringDecoderProvider.cs</Link>
-    </Compile>
-    <Compile Include="..\Common\TypeSystem\Ecma\CachingMetadataStringDecoder.cs">
-      <Link>Ecma\CachingMetadataStringDecoder.cs</Link>
-    </Compile>
-    <Compile Include="..\Common\TypeSystem\Ecma\PrimitiveTypeProvider.cs">
-      <Link>Ecma\PrimitiveTypeProvider.cs</Link>
-    </Compile>
-    <Compile Include="..\Common\TypeSystem\IL\EcmaMethodIL.cs">
-      <Link>IL\EcmaMethodIL.cs</Link>
-    </Compile>
-    <Compile Include="..\Common\TypeSystem\IL\MethodIL.cs">
-      <Link>IL\MethodIL.cs</Link>
-    </Compile>
-    <Compile Include="..\Common\TypeSystem\IL\MethodILDebugView.cs">
-      <Link>IL\MethodILDebugView.cs</Link>
-    </Compile>
-    <Compile Include="..\Common\TypeSystem\IL\ILDisassembler.cs">
-      <Link>IL\ILDisassembler.cs</Link>
-    </Compile>
-    <Compile Include="..\Common\TypeSystem\IL\InstantiatedMethodIL.cs">
-      <Link>IL\InstantiatedMethodIL.cs</Link>
-    </Compile>
-    <Compile Include="..\Common\TypeSystem\IL\ILOpcode.cs">
-      <Link>IL\ILOpcode.cs</Link>
-    </Compile>
-    <Compile Include="..\Common\TypeSystem\IL\ILImporter.cs">
-      <Link>IL\ILImporter.cs</Link>
-    </Compile>
-    <Compile Include="..\Common\TypeSystem\IL\StackValueKind.cs">
-      <Link>IL\StackValueKind.cs</Link>
-    </Compile>
-    <Compile Include="..\Common\TypeSystem\Interop\InstantiatedType.Interop.cs">
-      <Link>Interop\InstantiatedType.Interop.cs</Link>
-    </Compile>
-    <Compile Include="..\Common\TypeSystem\Interop\MetadataType.Interop.cs">
-      <Link>Interop\MetadataType.Interop.cs</Link>
-    </Compile>
-    <Compile Include="..\Common\TypeSystem\Interop\MethodDesc.Interop.cs">
-      <Link>Interop\MethodDesc.Interop.cs</Link>
-    </Compile>
-    <Compile Include="..\Common\TypeSystem\Interop\FieldDesc.Interop.cs">
-      <Link>Interop\FieldDesc.Interop.cs</Link>
-    </Compile>
-    <Compile Include="..\Common\TypeSystem\Interop\MarshalAsDescriptor.cs">
-      <Link>TypeSystem\Interop\MarshalAsDescriptor.cs</Link>
-    </Compile>
-    <Compile Include="..\Common\System\Collections\Generic\ArrayBuilder.cs">
-      <Link>Utilities\ArrayBuilder.cs</Link>
-    </Compile>
-    <Compile Include="..\Common\TypeSystem\Common\LocalVariableDefinition.cs">
-      <Link>TypeSystem\Common\LocalVariableDefinition.cs</Link>
-    </Compile>
-    <Compile Include="..\Common\System\FormattingHelpers.cs">
-      <Link>..\Common\System\FormattingHelpers.cs</Link>
-    </Compile>
-    <Compile Include="..\Common\TypeSystem\Common\TypeSystemConstraintsHelpers.cs">
-      <Link>TypeSystem\Common\TypeSystemConstraintsHelpers.cs</Link>
-    </Compile>
-  </ItemGroup>
+  <Import Project="ILVerification.projitems" />
```

Comment:
In order to open this correctly in VS, you'll need to add a .shproj just how we do with System.Private.CoreLib:

https://github.com/dotnet/runtime/blob/master/src/libraries/System.Private.CoreLib/src/System.Private.CoreLib.Shared.shproj

Reply:
What do I get by doing that? As far as I can tell, the current VS is perfectly happy without it.

Reply:
Does it show all the cs files? Maybe it is just needed for different subtrees. I don't recall why we needed to do it for S.P.CoreLib. @ViktorHofer do you remember?

Reply:
> Does it show all the cs files? 

Yes: ![image](https://user-images.githubusercontent.com/6668460/83758732-5e0bfa00-a627-11ea-95c4-2f69a70c5d8f.png)

Reply:
Ok then disregard my comment. I’ll let @ViktorHofer explain why it was needed for S.P.CoreLib.

Reply:
the shproj for CoreLib was necessary as the projitems file is located in a different directory tree. VS refused to load without it.

Reply:
Awesome, so then I did remember sort of correctly after @jkotas told me it worked fine. Thanks @ViktorHofer for solving the mystery 😄